.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [
+    "es2015",
+    "stage-0"
+  ],
+  "plugins": [
+    "transform-async-to-generator",
+    "istanbul",
+    "transform-async-to-generator"
```

Comment:
I think this should go. Double declaration

Reply:
Hello @wizzy25 .. updated here https://github.com/sphereio/sphere-product-type-export/pull/22/commits/0a632e9f8e605d0a558865faec9e5308a357eb5e#diff-e56633f72ecc521128b3db6586074d2cL9